lib/facility_creator.rb

```diff
@@ -0,0 +1,50 @@
+require 'date'
+
+class FacilityCreator
+  def initialize
+    @new_facilities = []
+  end
+
+  def create_co_facilities(co_remote_data)
+    co_remote_data.each do |data|
+      facility = Facility.new({
+        name:data[:dmv_office],
+        address:"#{data[:address_li]}, #{data[:address__1]}, #{data[:city]}, #{data[:state]}, #{data[:zip]}",
+        phone:data[:phone],
+        services:data[:services_p],
+        collected_fees:0,
+        registered_vehicles:[]})
+      @new_facilities << facility
+    end
+    @new_facilities
+  end
+
+  def create_ny_facilities(ny_remote_data)
+    ny_remote_data.each do |data|
+      facility = Facility.new({
+        name:data[:office_name],
+        address:"#{data[:street_address_line_1]}, #{data[:city]}, #{data[:state]}, #{data[:zip_code]}",
+        phone:data[:public_phone_number],
+        services:[],
+        collected_fees:0,
+        registered_vehicles:[]})
+      @new_facilities << facility
+    end
+    @new_facilities
+  end
+
+  def create_mo_facilities(mo_remote_data)
+    mo_remote_data.each do |data|
+      facility = Facility.new({
+        name:data[:name],
+        address:"#{data[:address1]}, #{data[:city]}, #{data[:state]}, #{data[:zipcode]}",
+        phone:data[:phone],
+        services:[],
+        collected_fees:0,
+        registered_vehicles:[]})
+      @new_facilities << facility
+    end
+    @new_facilities
+  end
+end
```

Comment:
I think there might have been a little confusion surrounding the expectation for iteration 3. It was intended for you to just have one method that would behave differently depending on the data that it got sent from either the CO, NY or MO endpoints. Currently, you have 3 different methods within this class.

The code overall is great and each method does what it is supposed to do but doesn't quite follow the specification.

One last thing, since we are instantiating new `Facility` objects we should `require` in the Facility class into this file.